pkg/controllers/clusterresourcebindingwatcher/watcher.go

```diff
@@ -33,6 +35,14 @@ type Reconciler struct {
 // Reconcile reconciles the clusterResourceBinding.
 func (r *Reconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
 	bindingRef := klog.KRef("", req.Name)
+
+	startTime := time.Now()
+	klog.V(2).InfoS("Reconciliation starts", "clusterResourceBinding", bindingRef)
+	defer func() {
+		latency := time.Since(startTime).Milliseconds()
+		klog.V(2).InfoS("Reconciliation ends", "clusterResourceBinding", bindingRef, "latency", latency)
```

Comment:
```suggestion
		klog.V(2).InfoS("ClusterResourceBindingWatcher reconciliation ends", "clusterResourceBinding", bindingRef, "latency", latency)
```

please use unique log string so that we can create the dashboard based on the log.